### PR TITLE
EclipseAdoptium.Temurin.18.JDK version 18.0.2.101

### PR DESCRIPTION
**Pull request by VlaDexa**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by VlaDexa** [comment]
Also, this is a newer release than the existing ones, but the versioning will think that this is an older one. 
Apparently Eclipse maintainers decided that adding another number to their versione won't do any harm

File: manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.18.JDK
+PackageVersion: 18.0.2.1.1
```
Comment (ItzLevvie) [inline manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.installer.yaml:5]:
```suggestion
PackageVersion: 18.0.2.101
```
This must match Control Panel; you will also need to update the version folder to match PackageVersion.

File: manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.18.JDK
+PackageVersion: 18.0.2.1.1
```
Comment (ItzLevvie) [inline manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 18.0.2.101
```
This must match Control Panel; you will also need to update the version folder to match PackageVersion.

File: manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.18.JDK
+PackageVersion: 18.0.2.1.1
```
Comment (ItzLevvie) [inline manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.yaml:5]:
```suggestion
PackageVersion: 18.0.2.101
```
This must match Control Panel; you will also need to update the version folder to match PackageVersion.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**